crates/modelardb_server/src/storage/data_transfer.rs

```diff
@@ -190,14 +198,29 @@ impl DataTransfer {
         );
 
         // Merge the files and transfer them to the remote object store.
```

Comment:
Maybe extend the comment to make it more clear how `merge_compressed_apache_parquet_files()` transfer the merged file.

Reply:
Fixed.